### DISCUSSION TITLE
Auffälligeres Erscheinungsbild der Hilfe-Funktion "?"

### DISCUSSION THREAD

**Discussion by faulgurke** [e-netze/webgis-community#37] [Ideas]
Wäre es nicht zweckmäßig, die Hilfe im Toolbar ("?") etwas auffälliger zu gestalten, bzw. mit einem Tooltip zu versehen?

**Comment by gis-eni** [comment]
Das Fragezeichen ist halt eher üblich, bei Verlinkungen zur Hilfe. Einen Tooltip braucht man dazu meiner Meinung nach nicht, es recht schon, wenn sich der Mauszeiger ändert.

Dir steht natürlich offen die Stylessheets entsprechend anzupassen um das Erscheinungsbild zu ändern, um den Button "prominenter" zu machen.
